db/schema.rb

```diff
@@ -0,0 +1,61 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 20_230_712_081_556) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension 'plpgsql'
+
+  create_table 'comments', force: :cascade do |t|
+    t.integer 'author_id'
+    t.integer 'post_id'
+    t.string 'text'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['author_id'], name: 'index_comments_on_author_id'
+    t.index ['post_id'], name: 'index_comments_on_post_id'
+  end
+
+  create_table 'likes', force: :cascade do |t|
+    t.integer 'author_id'
+    t.integer 'post_id'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['author_id'], name: 'index_likes_on_author_id'
+    t.index ['post_id'], name: 'index_likes_on_post_id'
+  end
+
+  create_table 'posts', force: :cascade do |t|
+    t.integer 'author_id'
+    t.string 'title'
+    t.text 'text'
+    t.integer 'comments_counter'
+    t.integer 'likes_counter'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['author_id'], name: 'index_posts_on_author_id'
+  end
+
+  create_table 'users', force: :cascade do |t|
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.string 'Name'
+    t.string 'Photo'
+    t.text 'Bio'
+    t.integer 'PostsCounter'
```

Comment:
- [ ] Following Ruby conventions we should use snake_casing and lower case for our table column names. Kiindly run new migrations and rename these column names.

  ```
  name, photo, bio, posts_counter
  ```

Reply:
Thanks so much @TedLivist ted